docs/src/ex/0_Introduction.md

```diff
@@ -0,0 +1,178 @@
+
+# Probabilistic Programming with Turing
```

Comment:
This seems like a slightly odd title for this section. I mean, what else is the tutorials section going to be on? Possibly consider "Coin-Flipping"

Reply:
I wanted this to be fairly broad as it is an introductory piece, mostly so people who are new to the ecosystem can go there immediately to get a handle on the basics. Perhaps "Introduction to Turing" or something similar?

Reply:
"Introduction to Turing" is way better imho. Perhaps just "An Introduction"?